x-pack/filebeat/modules.d/zeek.yml.disabled

```diff
@@ -0,0 +1,21 @@
+# Module: zeek
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-zeek.html
```

Comment:
Should we use `/current/` instead of `/master/` here?

Reply:
I think these headers are generated based on the [documentation branch](https://github.com/elastic/beats/blob/master/libbeat/docs/version.asciidoc#L2) which is master so this is correct.